libraries/libqtpas/libqtpas.SlackBuild

```diff
@@ -48,18 +48,21 @@
 
 # 07/10/2024: Updated to version 3.4
 
+# 31/01/2025: updated to version 3.8
+
 ########################################################################################
 
 
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=libqtpas
-VERSION=${VERSION:-3.4}
+VERSION=${VERSION:-3.8}
 BUILD=${BUILD:-1}
 TAG=${TAG:-_SBo}
 PKGTYPE=${PKGTYPE:-tgz}
 SRCNAM=${SRCNAM:-lazarus}
-SRCVER=${SRCVER:-3.6}
+SRCVER=${SRCVER:-3.8-0}
+nprocr=$(expr $(nproc) + 1)
```

Comment:
should respect MAKEFLAGS instead

Reply:
https://github.com/SlackBuildsOrg/slackbuilds/blob/cf3062747a175e70103f4c67e6779607db356e63/misc/rmlint/rmlint.SlackBuild#L86-L90